dashboards/dev-platform/pipelines-list.csv

```diff
@@ -1,3 +1,220 @@
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-dcmaw-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-drv-license-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-err-test-1-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-err-test-2-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-experian-kbv-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-f2f-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-fraud-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-hmrc-kbv-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-kbv-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-nino-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-passport-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,cri-toy-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,dcmaw-async-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,dns-build
+core-team,ipv-core@digital.cabinet-office.gov.uk,dns-integration
+core-team,ipv-core@digital.cabinet-office.gov.uk,dns-production
+core-team,ipv-core@digital.cabinet-office.gov.uk,dns-staging
+core-team,ipv-core@digital.cabinet-office.gov.uk,evcs-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,lambdanator
+core-team,ipv-core@digital.cabinet-office.gov.uk,orch-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,rum
+core-team,ipv-core@digital.cabinet-office.gov.uk,ticf-stub
+core-team,ipv-core@digital.cabinet-office.gov.uk,update-ecs-env-vars
+core-team,ipv-core@digital.cabinet-office.gov.uk,upload-assets
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,dl-cri-api-v1
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,dl-cri-front
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,fraud-cri-api-v1
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,fraud-cri-front
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,ipv-cri-passport-api
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,ipv-cri-passport-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,address-cri-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,bav-ipv-thirdparty-stub
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,check-hmrc-cri-api
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,check-hmrc-cri-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,common-cri-api
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,dl-third-party-stubs
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,fraud-third-party-stubs
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,kbv-cri-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,kbv-hmrc-cri-api
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,kbv-hmrc-cri-front
```

Comment:
@gds-jbentham Do you want the KBV HMRCs as Lime team?

Reply:
Changed. Thanks